config/babel/babel.config.js

```diff
@@ -2,36 +2,46 @@ module.exports = {
   plugins: [
     '@babel/plugin-proposal-class-properties',
     '@babel/plugin-proposal-object-rest-spread',
-    '@babel/plugin-transform-runtime',
+    ['@babel/plugin-transform-runtime', {
+      corejs: 3,
+    }],
     [
       'babel-plugin-styled-components',
       { ssr: false, displayName: false, namespace: 'sc' },
     ],
     ['babel-plugin-transform-imports', {
+      '@fortawesome/free-solid-svg-icons': {
+        transform: (importName) => `@fortawesome/free-solid-svg-icons/${importName}`,
+        preventFullImport: true,
+        skipDefaultConversion: true,
+      },
       '@fortawesome/pro-duotone-svg-icons': {
-        transform: (importName) => `@fortawesome/pro-duotone-svg-icons/${importName}`,
         preventFullImport: true,
         skipDefaultConversion: true,
+        transform: (importName) => `@fortawesome/pro-duotone-svg-icons/${importName}`,
       },
       '@fortawesome/pro-light-svg-icons': {
-        transform: (importName) => `@fortawesome/pro-light-svg-icons/${importName}`,
         preventFullImport: true,
         skipDefaultConversion: true,
+        transform: (importName) => `@fortawesome/pro-light-svg-icons/${importName}`,
       },
       '@fortawesome/pro-regular-svg-icons': {
-        transform: (importName) => `@fortawesome/pro-regular-svg-icons/${importName}`,
         preventFullImport: true,
         skipDefaultConversion: true,
+        transform: (importName) => `@fortawesome/pro-regular-svg-icons/${importName}`,
       },
       '@fortawesome/pro-solid-svg-icons': {
-        transform: (importName) => `@fortawesome/pro-solid-svg-icons/${importName}`,
         preventFullImport: true,
         skipDefaultConversion: true,
+        transform: (importName) => `@fortawesome/pro-solid-svg-icons/${importName}`,
       },
     }],
   ],
   presets: [
-    '@babel/preset-env',
+    ['@babel/preset-env', {
+      corejs: '3',
+      useBuiltIns: 'entry',
+    }],
```

Comment:
I based changes in our other libraries on this comment:

https://github.com/babel/babel/issues/9853#issuecomment-619587386

> If you are authoring a library, use only @babel/transform-runtime with corejs option plus @babel/runtime-corejs3 as dependency, and @babel/preset-env for syntax transpilation with useBuiltIns: false.

So this should be:

```
['@babel/preset-env', {
  useBuiltIns: false,
}],
```